test/regressions/site/src/tests/Input/Inputs.js

```diff
@@ -30,29 +35,35 @@ export default class Inputs extends Component {
     const classes = this.context.styleManager.render(styleSheet);
 
     return (
-      <div className={classes.container}>
-        <Input
-          defaultValue="Hello world"
-          className={classes.input}
-        />
-        <Input
-          placeholder="Placeholder"
-          className={classes.input}
-        />
-        <Input
-          value="Disabled"
-          className={classes.input}
-          disabled
-        />
-        <Input
-          error
-          value="Error"
-          className={classes.input}
-        />
+      <div>
+        <div className={classes.container}>
+          <Input
+            value="Hello world"
+            className={classes.input}
+          />
+          <Input
+            placeholder="Placeholder"
+            className={classes.input}
+          />
+          <Input
+            value="Disabled"
+            className={classes.input}
+            disabled
+          />
+          <Input
+            error
+            value="Error"
+            className={classes.input}
+          />
+          <Input
+            value="Focused"
+            ref={(c) => { this.focusInput = c; }}
+            className={classes.input}
+          />
+        </div>
         <Input
-          value="Focused"
-          ref={(c) => { this.focusInput = c; }}
-          className={classes.input}
+          value="Large input"
+          className={classNames(classes.input, classes.large)}
```

Comment:
That won't work, the `<Input />` do not grow by default. Here is the output with what you are suggesting:
![capture d ecran 2017-02-09 a 21 12 27](https://cloud.githubusercontent.com/assets/3165635/22801293/9007fffc-ef0c-11e6-8198-67a7598ac0fd.png)

Here is the same output by using `<input />` intead of `<Input />`.

![capture d ecran 2017-02-09 a 21 14 02](https://cloud.githubusercontent.com/assets/3165635/22801325/b4b1d8dc-ef0c-11e6-97be-0743d768ec3a.png)